releases/release-7.5.7.md

```diff
@@ -0,0 +1,172 @@
+---
+title: TiDB 7.5.7 Release Notes
+summary: 了解 TiDB 7.5.7 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 7.5.7 Release Notes
+
+发版日期：2025 年 9 月 4 日
+
+TiDB 版本：7.5.7
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v7.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v7.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v7.5.7#version-list)
+
+## 兼容性变更
+
+- 将系统变量 [`tidb_enable_historical_stats`](https://docs.pingcap.com/zh/tidb/v7.5/system-variables/#tidb_enable_historical_stats) 的默认值从 `ON` 修改为 `OFF`，即默认关闭历史统计信息，避免潜在的稳定性问题 [#53048](https://github.com/pingcap/tidb/issues/53048) @[hawkingrei](https://github.com/hawkingrei)
+- TiKV 废弃以下配置项，并由新的 [`gc.auto-compaction`](/tikv-configuration-file.md#gc.auto-compaction) 配置项替代，用于控制自动 compaction 行为 [#18727](https://github.com/tikv/tikv/issues/18727) @[v01dstar](https://github.com/v01dstar)
```

Comment:
@v01dstar Could you please review this release note and confirm whether the GitHub issue link is correct?